client/dashboard/me/active-subscriptions/constants.ts

```diff
@@ -0,0 +1,53 @@
+export const SubscriptionBillPeriod = {
```

Comment:
Team review: [nit] Consider naming constants in all caps. SubscriptionBillPeriod -> SUBSCRIPTION_BILL_PERIOD.

Reply:
I'm not sure. The existing constants like `DotcomFeatures` or `DotcomPlans` don't follow that pattern. I think I personally like camel case for the group.

Reply:
Ah yes, I forgot that. However, it does seem a bit inconsistent to me that the outer name uses SentenceCaps, while the properties do not 🙈